src/h5web/toolbar/controls/DomainSlider/DomainSlider.module.css

```diff
@@ -27,11 +30,17 @@
   flex: 1 1 0%;
   align-self: center;
   height: 0.5rem;
+  margin: 0 calc(var(--thumb-size-default) / 2); /* make track end in middle of thumbs */
   background-color: var(--secondary-dark-15);
   box-shadow: var(--btn-shadow-pressed) var(--btn-shadow-color);
   border-radius: 0.25rem;
 }
 
+.track[data-automin] {
+  /* Make track end in middle of thumbs when auto-scaling min and/or max */
+  margin: 0 calc((var(--thumb-size-auto) - 0.125rem) / 2); /* MAGIC NUMBER */
```

Comment:
Let's be honest, I'm not 100% sure why this works. So ... better make it clear with a comment. Again, if you find better, please be my guest 😂